keanu-project/src/main/java/io/improbable/keanu/util/io/ProtobufSaver.java

```diff
@@ -15,6 +15,7 @@
 import io.improbable.keanu.vertices.intgr.IntegerVertex;
 import io.improbable.mir.KeanuSavedBayesNet;
 import io.improbable.mir.SavedBayesNet;
+import kotlin.Pair;
```

Comment:
I'd prefer not to include Kotlin stuff in the native Java code if at all possible - so could we just create our own Pair class?

Reply:
Okay I'll try to use a different `Pair` class. I think there are a few options

Reply:
@christophernorth what about `org.apache.commons.math3.util.Pair`?